cmd/backup-manager/app/util/util.go

```diff
@@ -395,6 +400,313 @@ func GetBRMetaData(ctx context.Context, provider v1alpha1.StorageProvider) (*kvb
 	return backupMeta, nil
 }
 
+// CalcBackupSizeFromBackupmeta get backup size from backup meta
+func CalcBackupSizeFromBackupmeta(ctx context.Context, provider v1alpha1.StorageProvider) (int64, error) {
```

Comment:
Can we migrate it to `snapshotter` package? Because every type of volume snapshot on different storage provider all need this function.

Reply:
checked the code, it looks `snapshotter` do backup meta prepare and commit, it does not relate to any cloud action.

anyway, we need an abstract interface for the general cloud platform, let's do this after GCP support volume-snapshot

Reply:
refactor in https://github.com/pingcap/tidb-operator/pull/4819/commits/fc18e21a421dee08cf3257099b503047e5d552a7